src/component/Calculator.js

```diff
@@ -1,67 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Calculator.css';
 import calculate from '../logic/calculate';
 
-class Calculator extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      next: null,
-      operation: null,
-    };
-  }
+function Calculator() {
+  const [state, setState] = useState({
```

Comment:
- [ ] Kindly convert your function into ES6 version as shown below :arrow_down: to follow the JavaScript best practices :+1:

**Your Function :red_circle:**
```
function Calculator() {
  ###
}

```
**ES6 Function :green_circle:**
```
const Calculator = () => {
  ###
}
```